src/helpers/Sep9Fields.ts

```diff
@@ -0,0 +1,214 @@
+  },
+  {
+    name: "address_country_code",
+    type: Sep9FieldType.COUNTRY_CODE,
+    description: "Country code for current address",
+  },
+  {
+    name: "state_or_province",
+    type: Sep9FieldType.STRING,
+    description: "Name of state/province/region/prefecture",
+  },
+  {
+    name: "city",
+    type: Sep9FieldType.STRING,
+    description: "Name of city/town",
+  },
+  {
+    name: "postal_code",
+    type: Sep9FieldType.STRING,
+    description: "Postal or other code identifying user's locale",
+  },
+  {
+    name: "address",
+    type: Sep9FieldType.STRING,
+    description:
+      "Entire address (country, state, postal code, street address, etc...) as a multi-line string",
+  },
+  {
+    name: "mobile_number",
+    type: Sep9FieldType.PHONE_NUMBER,
+    description: "Mobile phone number with country code, in E.164 format",
+  },
+  {
+    name: "email_address",
+    type: Sep9FieldType.STRING,
+    description: "Email address",
+  },
+  {
+    name: "birth_date",
+    type: Sep9FieldType.DATE,
+    description: "Date of birth, e.g. 1976-07-04",
+  },
+  {
+    name: "birth_place",
+    type: Sep9FieldType.STRING,
+    description: "Place of birth (city, state, country; as on passport)",
+  },
+  {
+    name: "birth_country_code",
+    type: Sep9FieldType.COUNTRY_CODE,
+    description: "ISO Code of country of birth",
+  },
+  {
+    name: "bank_account_number",
+    type: Sep9FieldType.STRING,
+    description: "Number identifying bank account",
+  },
+  {
+    name: "bank_number",
+    type: Sep9FieldType.STRING,
+    description:
+      "Number identifying bank in national banking system (routing number in US)",
+  },
+  {
+    name: "bank_phone_number",
+    type: Sep9FieldType.STRING,
+    description: "Phone number with country code for bank",
+  },
+  {
+    name: "bank_branch_number",
+    type: Sep9FieldType.STRING,
+    description: "Number identifying bank branch",
+  },
+  {
+    name: "tax_id",
+    type: Sep9FieldType.STRING,
+    description:
+      "Tax identifier of user in their country (social security number in US)",
+  },
+  {
+    name: "tax_id_name",
+    type: Sep9FieldType.STRING,
+    description: "Name of the tax ID (SSN or ITIN in the US)",
+  },
+  {
+    name: "occupation",
+    type: Sep9FieldType.NUMBER,
+    description: "Occupation ISCO code",
+  },
+  {
+    name: "employer_name",
+    type: Sep9FieldType.STRING,
+    description: "Name of employer",
+  },
+  {
+    name: "employer_address",
+    type: Sep9FieldType.STRING,
+    description: "Address of employer",
+  },
+  {
+    name: "language_code",
+    type: Sep9FieldType.LANGUAGE_CODE,
+    description: "Primary language",
+  },
+  {
+    name: "id_type",
+    type: Sep9FieldType.STRING,
+    description: "passport, drivers_license, id_card, etc...",
+  },
+  {
+    name: "id_country_code",
+    type: Sep9FieldType.COUNTRY_CODE,
+    description:
+      "country issuing passport or photo ID as ISO 3166-1 alpha-3 code",
+  },
```

Comment:
NIT: was it intentional that these two descriptions start with a lowercase letter?

Reply:
Good catch! I copied/pasted it from SEP-9 and forgot to adjust these descriptions.
Fixed in https://github.com/stellar/stellar-demo-wallet/pull/175/commits/6c52210215d87aa466765df092ef4866e5c5f899.